src/client/index.ts

```diff
@@ -24,26 +24,28 @@ export function ViteSSG(
     rootContainer = '#app',
     hydration = false,
   } = options
-  const isClient = typeof window !== 'undefined'
+  const isClient = !!import.meta.env.SSR
 
   async function createApp(client = false, routePath?: string) {
-    const app = client && !hydration
-      ? createClientApp(App)
-      : createSSRApp(App)
+    const app = import.meta.env.SSR
```

Comment:
uhmm

Reply:
ok, it is fine, once the client is built then the code is the same, but how about `unhead` with `client && !hydration`?

Reply:
useHead is agnostic